sites/website/src/docs/community/engineering-process/issue-management.md

```diff
@@ -0,0 +1,120 @@
+
+## Automation
+
+We rely on some automation to help us with this process. New issues are tagged with `status:triage` so that we do not miss them during our triage process. If issues have no activity for a prolonged period of time, a bot will mark them as `warning:stale` and will close the issue in the following weeks if there continues to be no activity.
+
+## Policies
+
+We have a lot of issue traffic to manage, so we have some policies in place to help us do that. This is a brief summary of some of the policies we have in place and the justification for them.
+
+### Commenting on Closed Issues
+
+In general, we recommend you open a new issue if you have a bug, feature request, or question to discuss. If you find a closed issue that is related, open a *new issue* and link to the closed issue rather than posting on the closed issue. Closed issues don't appear in our triage process, so only the people who have been active on the original thread will be notified of your comment. A new issue will get more attention from the team.
+
+*In general* we don't mind getting duplicate issues. It's easier for us to close duplicate issues than to discuss multiple root causes on a single issue! We may close your issue as a duplicate if we determine it has the same root cause as another. Don't worry! It's not a waste of our time!
+
+### Needs Information
+
+If a contributor reviews an issue and determines that more information is needed from the author, they will post a comment requesting that information and apply the `status:needs-information` label. This label indicates that the author needs to post a response in order for us to continue investigating the issue.
+
+If the author does not post a response within **7 days**, the issue may be closed. If the author responds within **7 days** after the issue is closed, the issue will be  re-opened. We recognize that you may not be able to respond immediately to our requests, we're happy to hear from you whenever you're able to provide the new information.
+
+### PR: Needs Author Input
+
+Similar to the `status:needs-information` process above, PRs also require author input from time to time. When a member of our team asks for some follow-up changes from the author, we mark these PRs with `status:needs-author-input` label. After doing so, we expect the author to respond within 14 days.
+If the author does not post a response or updates the PR within **14 days**, the issue will be closed. If the author responds within **7 days** after the issue is closed, the issue will be re-opened. We recognize that you may not be able to respond immediately to our requests, we're happy to hear from you whenever you're able to provide the new information.
```

Comment:
```suggestion
If the author does not post a response or updates the PR within **14 days**, the issue may be closed. If the author responds within **7 days** after the issue is closed, the issue will be re-opened. We recognize that you may not be able to respond immediately to our requests, we're happy to hear from you whenever you're able to provide the new information.
```
Above in the "Needs Information" section you use "may" instead of "will" when referencing closing an issue - using "will" struck me as a hard commitment we were making that maybe wasn't intentional?

Reply:
I like the softer approach you recommend. In practice we probably won't do that exactly unless later we introduce automation for this. So, I agree "may" is better.